srcpkgs/intellij-idea-ultimate-edition/template

```diff
@@ -1,90 +1,46 @@
 # Template file for 'intellij-idea-ultimate-edition'
 pkgname=intellij-idea-ultimate-edition
-version=2021.3.3
-revision=2
-archs="i686 x86_64"
-depends="giflib libXtst jetbrains-jdk-bin lldb-devel"
-short_desc="Most intelligent Java IDE"
+version=2024.1.2
+revision=1
+archs="x86_64"
+depends="virtual?java-environment giflib libXtst hicolor-icon-theme"
```

Comment:
does this work with any jdk (8, 11, 17) or does it need at least some version?

Reply:
Jetbrains recommends JDK17, but it should work with any supported version